### ISSUE TITLE
Crash with `Error: failed to unpack child-ids tarball`

### ISSUE THREAD

**Issue by leplatrem** [mozilla/remote-settings-ondemand-crashes#2]
```
Get last modified time from https://remote-settings-dev.allizom.org/v1/buckets/main-workspace/collections/crash-reports-ondemand/records
Get crash ids from https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/mozilla.v2.process-top-crashes.latest.process-pings/artifacts/public/crash-ids.tar.gz
Unpacking child ids tarball
tar exited with error code 2
Error: failed to unpack child-ids tarball
    at update (file:///app/update_remote_settings_records.mjs:83:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
```

**Comment by leplatrem** [comment]
Same same two runs in a row 

<img width="1443" alt="Screenshot 2024-11-08 at 16 03 06" src="https://github.com/user-attachments/assets/08c213fc-c16c-4199-9714-45c9de255cab">

**Comment by afranchuk** [comment]
Closed by 8c0dd5a21e167a1931a4939bc019ae4713177588.
